examples/docker-compose-counter-v3.yaml

```diff
@@ -0,0 +1,20 @@
+version: "3"
+
+services:
+
+ web:
+  image: tuna/docker-counter23
+  ports:
+    - "5000:5000"
+  deploy:
+    restart_policy:
+      condition: any
+  labels:
+    kompose.service.type: NodePort
+
+ redis:
+  image: redis:3.0
+  deploy:
+     replicas: 1
```

Comment:
what's the point of putting replicas: 1 when it's not a cluster (master / slave)? and 1 is default anyways?